package.json

```diff
@@ -42,26 +39,24 @@
   },
   "homepage": "https://github.com/libp2p/js-libp2p-keychain#readme",
   "dependencies": {
-    "async": "^2.6.2",
-    "err-code": "^1.1.2",
-    "interface-datastore": "~0.6.0",
-    "libp2p-crypto": "~0.16.1",
+    "err-code": "^2.0.0",
+    "interface-datastore": "^0.7.0",
+    "libp2p-crypto": "^0.17.0",
     "merge-options": "^1.0.1",
-    "node-forge": "~0.7.6",
-    "pull-stream": "^3.6.9",
+    "node-forge": "^0.8.5",
+    "promisify-es6": "^1.0.3",
```

Comment:
Should this be a dev dep? It's added by this PR but only used in `test/node.js` that I can see.